app/Dockerfile

```diff
@@ -37,40 +37,40 @@ RUN apt-get install -y \
 
 # Install MIRTK/SVRTK
 RUN git clone https://github.com/SVRTK/MIRTK.git \
-    && mkdir -p /home/MIRTK/Packages/SVRTK
-RUN git clone https://github.com/SVRTK/SVRTK.git /home/MIRTK/Packages/SVRTK \
-    && mkdir -p /home/MIRTK/build \
-	&& cd /home/MIRTK/build \
+    && mkdir -p /bin/MIRTK/Packages/SVRTK
+RUN git clone https://github.com/SVRTK/SVRTK.git /bin/MIRTK/Packages/SVRTK \
+    && mkdir -p /bin/MIRTK/build \
+	&& cd /bin/MIRTK/build \
 	&& cmake -D WITH_TBB="ON" -D MODULE_SVRTK="ON" .. \
 	&& make -j
 
-ENV PATH="$PATH:/home/MIRTK/build/bin:/home/MIRTK/build/lib/tools"
+ENV PATH="$PATH:/bin/MIRTK/build/bin:/bin/MIRTK/build/lib/tools"
 
 WORKDIR /var/monai
 
 
-# Add 3D UNet model weights & execution script
+# Add MONAI 3D UNet model weights & execution script
 
 WORKDIR /home
 
 # Setup 3D UNet models
-RUN git clone https://github.com/SVRTK/Segmentation_FetalMRI.git --branch svrtk-docker-gpu-0.10 --single-branch /home/Segmentation_FetalMRI
+RUN git clone https://github.com/SVRTK/auto-proc-svrtk.git /home/auto-proc-svrtk
```

Comment:
Most of the errors are to do with this line currently. You've changed the location of the recon Shell script, but not updated the rest of the Dockerfile and Python code to reflect the new script location, so... see other comments where I can see you need to make changes.

Reply:
i fixed paths and was able to run monai-deploy run ghcr.io/svrtk/aide-svrtk/map:0.2.1 input/ output/